example/src/pages/MapBaseLayers.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Select, ViewerContainer } from '@datapunt/asc-ui'
+import { Map } from '@datapunt/react-maps'
+import Controls from '../../../src/components/Zoom'
+import BaseLayer from '../../../src/components/BaseLayer'
+import Scale from '../../../src/components/Scale'
+import {
+  MapLayer,
+  DEFAULT_AMSTERDAM_LAYERS,
+  DEFAULT_AMSTERDAM_MAPS_OPTIONS,
+  AERIAL_AMSTERDAM_LAYERS,
+} from '../../../src/constants'
+
+const StyledMap = styled(Map)`
+  width: 100%;
+  height: 100vh;
+`
+
+const StyledViewerContainer = styled(ViewerContainer)`
+  z-index: 400;
+`
+
+const MapBaseLayers: React.FC = () => {
+  const [baseLayer, setBaseLayer] = useState<string>(
```

Comment:
It's not required to pass the type here as it can be inferred from the default value passed into it.